kubernetes/compatibility/features.go

```diff
@@ -17,3 +17,8 @@ func (v Version) FeatureFlagSeccompDefaultEnabledByDefault() bool {
 	// see https://github.com/kubernetes/kubernetes/pull/110805
 	return semver.Version(v).GTE(semver.Version{Major: 1, Minor: 25})
 }
+
+func (v Version) SupportsV1KubeSchedulerConfiguration() bool {
```

Comment:
I'm not sure if we return a `bool` or the actual version as string here

Reply:
I think actual version might be easier going forward?

Reply:
thanks, makes more sense